apps/src/templates/lessonOverview/ResourceList.jsx

```diff
@@ -83,6 +91,18 @@ export default class ResourceList extends Component {
     );
   };
 
+  sendLinkVisitedEvent = (resource, visitType) => {
+    analyticsReporter.sendEvent(EVENTS.LESSON_RESOURCE_LINK_VISITED_EVENT, {
+      resourceId: resource.id,
+      resourceName: resource.name,
+      resourceAudience: resource.audience,
+      resourceType: resource.type,
+      resourceUrl: resource.url,
+      resourceDownloadUrl: resource.download_url,
+      visitType: visitType
```

Comment:
@dju90 I'm just making up all these keys, since I wasn't sure what would be useful here. We don't have access to the lesson data in this component, unfortunately. Is there anything in here you don't want, or anything else I should try to add in?